src/Microsoft.VisualStudio.ProjectSystem.VisualBasic/ProjectSystem/LanguageServices/VisualBasicCommandLineParserProvider.cs

```diff
@@ -13,11 +14,21 @@ public VisualBasicCommandLineParserProvider()
         {
         }
 
-        [Export(typeof(CommandLineParser))]
+        [Export(typeof(IParseCommandLineArguments))]
         [AppliesTo(ProjectCapability.VisualBasic)]
-        public CommandLineParser Parser
+        public IParseCommandLineArguments Parser
         {
-            get { return VisualBasicCommandLineParser.Default; }
+            get { return new VisualBasicParseCommandLineArguments(); }
+        }
+
+        internal class VisualBasicParseCommandLineArguments : IParseCommandLineArguments
```

Comment:
Instead of continuing with the Export property approach, turn this inner class into the actual thing that is exporting.